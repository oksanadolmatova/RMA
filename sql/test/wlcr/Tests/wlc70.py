from __future__ import print_function

try:
    from MonetDBtesting import process
except ImportError:
    import process
import os, sys

dbfarm = os.getenv('GDK_DBFARM')
tstdb = os.getenv('TSTDB')

if not tstdb or not dbfarm:
    print('No TSTDB or GDK_DBFARM in environment')
    sys.exit(1)

#clean up first
dbname = tstdb
dbnameclone = tstdb + '-clone'

s = process.server(dbname = dbname, stdin = process.PIPE, stdout = process.PIPE, stderr = process.PIPE)

c = process.client('sql', dbname = dbname, stdin = process.PIPE, stdout = process.PIPE, stderr = process.PIPE)

#continue logging
cout, cerr = c.communicate('''\
create table tmp70(i int, s string);
insert into tmp70 values(1,'hello'), (2,'world');
select * from tmp70;
''')

sout, serr = s.communicate()

sys.stdout.write(sout)
sys.stdout.write(cout)
sys.stderr.write(serr)
sys.stderr.write(cerr)
