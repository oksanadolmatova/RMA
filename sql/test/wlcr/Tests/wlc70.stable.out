stdout of test 'wlc70` in directory 'sql/test/wlcr` itself:


# 12:34:11 >  
# 12:34:11 >  "/usr/bin/python2" "wlc70.py" "wlc70"
# 12:34:11 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_test_wlcr', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:38889/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-1561/.s.monetdb.38889
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
#create table tmp2(i int, s string);
#insert into tmp2 values(1,'hello'), (2,'world');
[ 2	]
#select * from tmp70;
% sys.tmp70,	sys.tmp70 # table_name
% i,	s # name
% int,	clob # type
% 1,	5 # length
[ 1,	"hello"	]
[ 2,	"world"	]

# 12:34:12 >  
# 12:34:12 >  "Done."
# 12:34:12 >  

